tensorflow/examples/speech_commands/test_streaming_accuracy.py

```diff
@@ -0,0 +1,126 @@
+import logging
+import soundfile as sf
+import tensorflow as tf
+import numpy as np
+from recognize_commands import RecognizeCommands, RecognizeResult
+from accuracy_utils import StreamingAccuracyStats
+
+
+# Reads a model graph definition from disk, and creates a default graph object
+
+def load_graph(mode_file):
+    graph = tf.Graph()
+    with graph.as_default():
+        od_graph_def = tf.GraphDef()
+        with tf.gfile.GFile(mode_file, 'rb') as fid:
+            serialized_graph = fid.read()
+            od_graph_def.ParseFromString(serialized_graph)
+            tf.import_graph_def(od_graph_def, name='')
+    return graph
+
+
+# Takes a file name, and loads a list of labels from it, one per line, and
+# returns a vector of strings.
+
+def read_label_file(file_name):
+    label_list = []
+    with open(file_name, 'r') as f:
+        lines = f.readlines()
+        for line in lines:
+            label_list.append(line.strip())
+    return label_list
+
+# Takes a file name, loads a piece of wav from it, and return sample_rate and numpy data of np.float64
+
+def read_wav_file(file_name):
+    data, sample_rate = sf.read(file_name,dtype='float32')
+    return sample_rate, data
+
+
+def main():
+    logging.basicConfig(level=logging.INFO)
+
+    wav_file = "/home/yuan/datasets/speech_dataset/generated_wav_and_label/streaming_test.wav"
```

Comment:
Can you take this in as a parameter (using absl flags perhaps). Same with everything else